src/modify.jl

```diff
@@ -1,7 +1,10 @@
 ###### update ####################
 
 function update{T,N,D}(ta::TimeArray{T,N,D}, tstamp::D, val::Vector{T})
-    if tstamp < maximum(ta.timestamp)
+    if length(ta) == 0
+        t    = vcat(tstamp)
```

Comment:
Why vcat? I think vec would be more appropriate, assuming that's what you're trying to do.